couchbase/search/v1/search.proto

```diff
@@ -241,7 +240,36 @@ message SearchQueryRequest {
 }
 
 message SearchQueryResponse {
-  repeated bytes hits = 1;
+
+  message SearchQueryRow {
+    string id = 1;
+    double score = 2;
+    string index = 3;
+    bytes explanation = 4;
+    bytes locations = 5;
+    map<string, bytes> fields = 6;
+    map<string, Fragment> fragments = 7;
+  }
+
+  message Fragment {
+    repeated string content = 1;
+  }
+
+  message Facet {
```

Comment:
This is missing the date_range, numeric_range, term_range fields.

Reply:
I put those in the enum FaceResultType below. I don't remember where I found this though, it's not specified in the RFC strangely. It was probably in the SDK instead https://github.com/couchbaselabs/sdk-rfcs/blob/master/rfc/0052-sdk3-full-text-search.md#searchfacetresult

Reply:
FacetResultType is just an enum? The ranges are lists, see https://docs.couchbase.com/server/current/fts/fts-search-response-facets.html